requirements-ci.txt

```
@@ -0,0 +1,15 @@
+numpy
+scipy
+sympy
+tensorflow
+tensorboard
+networkx
+git+https://github.com/XanaduAI/blackbird@master#egg=quantum-blackbird
```

We should be able to use the released version of Blackbird now.
```suggestion
quantum-blackbird
```